src/tir/transforms/inject_virtual_thread.cc

```diff
@@ -476,11 +476,6 @@ class VirtualThreadInjector : public StmtMutator {
   }
 };
 
-Stmt InjectVirtualThread(Stmt stmt) {
```

Comment:
why are we removing this?

Reply:
Because it's never used :-) Looks like someone forgot to remove it.